lib/matplotlib/transforms.py

```diff
@@ -694,8 +751,8 @@ def __init__(self, points, **kwargs):
         Notes
         -----
         If you need to create a `Bbox` object from another form
-        of data, consider the static methods :meth:`unit`,
-        :meth:`from_bounds` and :meth:`from_extents`.
+        of data, consider the static methods :meth:`from_bounds` and
```

Comment:
This note is rather redundant to your examples. IMHO it's enough to mention `from_bounds` and `from_extents` once.